src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,14 +1,13 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import lombok.Getter;
+
+@Getter
 public class CinemaHallRequestDto {
+    @Min(value = 0)
     private int capacity;
+    @NotNull(message = "Description may not be empty")
```

Comment:
the same with other messages
```suggestion
    @NotNull(message = "Description can't be null")
```